### ISSUE TITLE
Add option to remove author

### ISSUE THREAD

**Issue by sleeyax** [vweevers/remark-common-changelog#12]
When generating a new changelog entry, I'd like to omit the author of the commit from the end of each line. There doesn't seem to be an option to disable this yet as it just appends it here: https://github.com/vweevers/remark-common-changelog/blob/e741ff2f04028cfd347d92aa76ce26a39ab359bd/lib/get-changes.js#L46

**Comment by vweevers** [comment]
Yeah, we could do that. Are you willing and able to make a pull request?

Unrelated to the code - would you mind leaving feedback in https://github.com/vweevers/common-changelog/issues/9 describing your use case?

**Comment by sleeyax** [comment]
I'm willing but not able at the moment (not at my home office rn).
